### PR TITLE
Add Pentobi

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/enz/pentobi/issues/10
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build io.sourceforge.pentobi

File: io.sourceforge.pentobi.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+
+modules:
+  - name: pentobi
+    buildsystem: cmake
```
Comment (hfiguiere) [inline io.sourceforge.pentobi.yml:16]: Cmake-ninja doesn't work?
```suggestion
    buildsystem: cmake-ninja
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: io.sourceforge.pentobi.yml (hunk 1 above)
Comment (Mailaender) [reply io.sourceforge.pentobi.yml:16]: No it fails due to cyclic dependencies.

**Comment by enz** [comment]
Great. If you are interested in maintaining the flatpack config, I could add a link from the Pentobi homepage.

Also note that  PENTOBI_OPEN_HELP_EXTERNALLY is no longer neede with Pentobi >= 19.0 because it always uses an external browser (and it won't be needed in the future because Pentobi >= 21.0 will include a custom help viewer impementation, which neither needs QtWebView nor an external browser).

**Comment by Mailaender** [comment]
Removed the option. I think for the help to work you need a browser installed in the sandbox as well (untested) which is a bit unfortunate.

**Comment by barthalion** [comment]
bot, build io.sourceforge.pentobi

**Comment by barthalion** [comment]
Yeah, opening help definitely doesn't work. I know GNOME apps somehow handle this, but I don't remember how. Alternatively, help button could be changed to open an external URL.

**Pull request closed without merging**